interpreter/cling/lib/Interpreter/CIFactory.cpp

```diff
@@ -1243,6 +1244,9 @@ namespace {
     std::vector<const char*> argvCompile(argv, argv+1);
     argvCompile.reserve(argc+32);
 
+    bool debuggingEnabled = cling::utils::isTrue(std::getenv("CLING_DEBUG"));
```

Comment:
I'd recommend using `llvm::sys::Process::GetEnv`

Reply:
[`std::getenv`](https://en.cppreference.com/w/cpp/utility/program/getenv) is part of the C++ standard, what is the advantage of using `llvm::sys::Process::GetEnv` over that? Also, other parts of the same cpp file are using `getenv` as well.

Reply:
It handles utf8, etc. You are welcome to check out the implementation if you have more questions. Indeed, it may have other occurrences of other code elsewhere but we are reviewing this PR.

I don't think we should beat these topics to death as it usually goes with similar contributions. In general, Cling intends deep integration in LLVM and we want our code to look like as part of LLVM. If they have facilities for these things we use them with higher priority as there is usually a good reason even if we don't know it.

Reply:
cling uses `getenv` *exclusively* throughout the code. There are 15 occurrences of `getenv` in the code, and no occurrences of `GetEnv`. Even LLVM itself uses `getenv` more often than `llvm::sys::Process::GetEnv`. If you feel strongly about this, please create a pull request to replace occurrences of `getenv` with `llvm::sys::Process::GetEnv` for the rest of cling, and then I will update this pull request to use that as well. I don't want to break consistency to introduce `llvm::sys::Process::GetEnv` in a single place in cling where UTF-8 support is not necessary, as we only need to check for `"1"`, `"true"`, `"on"`, and `"yes"` as values.

Reply:
Ok, fair enough, I can update your code as well.